src/App.js

```diff
@@ -1,41 +1,21 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Routes, Route, Link } from "react-router-dom";
+import Navbar from "./components/Navbar";
+import Sidebar from "./components/Sidebar";
+import Searchbar from "./components/Searchbar";
+import NoteCard from "./components/NoteCard";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Navbar />
+      <div className="app-container pt-1">
+        <Sidebar />
+        <Searchbar />
+        <NoteCard />
+      </div>
+
+      <Routes>{/* <Route path="/" element={< />} /> */}</Routes>
```

Comment:
comments to be removed if not in use

Reply:
Will remove them at the end for sure. Thanks for pointing it out.